tasks/conf.yml

```diff
@@ -44,6 +44,12 @@
 - name: Set --http-port option
   template: src=http_port.j2 dest=/etc/marathon/conf/http_port
 
+- name: Set additional options
+  template: src=custom_option.j2 dest=/etc/marathon/conf/{{ item.name }}
```

Comment:
this looks a bit magic especially that custom_option.j2 has ``{{ item.value }}`` in it.

how about using copy module

```yaml
      copy:
        content: "{{ item.value }}"
        dest: "/etc/marathon/conf/{{ item.name }}"
```